nuget/bblanchon.PDFium.iOS.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>bblanchon.PDFium.iOS</id>
+    <version>$version$</version>
+    <authors>Benoit Blanchon</authors>
+    <owners>Benoit Blanchon</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="expression">Apache-2.0</license>
+    <licenseUrl>https://licenses.nuget.org/Apache-2.0</licenseUrl>
+    <projectUrl>https://github.com/bblanchon/pdfium-binaries</projectUrl>
+    <description>This package contains pre-compiled binaries of the PDFium library, an open-source library for PDF manipulation and rendering.</description>
+    <releaseNotes>$releaseNotes$</releaseNotes>
+    <copyright>Copyright © Benoit Blanchon 2017-2022</copyright>
+    <tags>PDFium PDF binaries library native Chromium</tags>
+    <repository type="git" url="https://github.com/bblanchon/pdfium-binaries.git" branch="$branch$" commit="$commit$" />
+  </metadata>
+
+  <files>
+    <!-- the native PDFium binaries -->
+    <!-- see list of available runtime identifiers -->
+    <!-- https://docs.microsoft.com/en-us/dotnet/core/rid-catalog -->
+    <file src="pdfium-ios-arm64/lib/GNUSparseFile.0/libpdfium.dylib" target="runtimes/ios-arm64/native/libpdfium.dylib" />
```

Comment:
iOS is not listed in [the RID catalog](https://docs.microsoft.com/en-us/dotnet/core/rid-catalog).
Is it really supported?

Reply:
It is! This article is outdated, unfortunatly.

You can find [all supported runtime identifiers here](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json).

Reply:
Excellent! Thanks.